test/basic.html

```diff
@@ -70,6 +70,14 @@
           });
         });
 
+        test('enableTransition(false) disables animations', function() {
+          collapse.enableTransition(false);
+          // this will trigger the animation logic
```

Comment:
nit: I think we have a secret agreement that comments should be at least capitalized (and maybe end in a period!). This doesn't always happen for older files, but new ones are pretty consistently like that (minus the punctuation, which is my personal grief, probably)
```js
// This is the best comment ever.
```